website/static/js/pages/project-settings-page.js

```diff
@@ -227,7 +227,13 @@ $(document).ready(function() {
       var unchecked = checkedOnLoad.filter('#selectAddonsForm input:not(:checked)');
 
       if(unchecked.length > 0 || checked.length > 0) {
-        return 'The changes on addon setting are not submitted!';
+          return 'The changes on addon setting are not submitted!';
+      }
+    /* Before closing the page, Check whether changes made to category, title or description are updated or not */
+      if ($('#titleID')[0].value !== (projectSettingsVM).titlePlaceholder  ||
+          $('#descriptionID')[0].value !== (projectSettingsVM).descriptionPlaceholder ||
+          $('#categoryID')[0].value !== (projectSettingsVM).categoryPlaceholder  ) {
+          return 'There are unsaved changes in your project section.';
```

Comment:
I would change `project section` to `project settings` to be consistent with other error message.  There we are calling them addon settings.